### ISSUE TITLE
resolution question

### ISSUE THREAD

**Issue by aaa2222339** [TencentARC/T2I-Adapter#79]
At present, the code only supports generating images with a resolution of 512 * 512. Can I generate images with a resolution of 512 * 768？

**Comment by MC-E** [comment]
Yes, it supports. You can try it on our online demos https://huggingface.co/Adapter.
